h2o-bindings/bin/custom/python/gen_deeplearning.py

```diff
@@ -24,14 +24,14 @@ def __init__(self, **kwargs):
 overrides = dict(
     initial_biases=dict(
         setter="""
-assert_is_type({pname}, None, [H2OFrame, None])
+assert_is_type({pname}, None, [None, str, H2OFrame])
```

Comment:
Isn't one of the `None`s superfluous here? Looks to me like `_check_type` checks `Union[None, Union[None, str, H2OFrame]]`. Same on the line 34.

Reply:
checktype should check `Union[None, List[Union[None, str, H2OFrame]]]`